apps/api/src/form/form.controller.ts

```diff
@@ -12,10 +27,26 @@ export class FormController {
   @ApiOperation({
     summary: 'Create a new record',
   })
+  @ApiBody({
+    schema: SwaggerSchema,
+  })
+  @UseInterceptors(ClassSerializerInterceptor)
   @ApiResponse({ status: HttpStatus.CREATED, type: EmptyResponse })
   @HttpCode(HttpStatus.CREATED)
   @Post()
-  async name(@Body() body: FormDTO): Promise<void> {
-    await this.formService.saveForm(body);
+  async name(@Body() body: FormDTO): Promise<FormEntity> {
```

Comment:
Do we want to send whole entity as response? I suggest, confirmation id is sufficient

Reply:
The entity contained in the RO could theoretically be used to fill info for the "Download a copy" feature (although that data should already be in browser memory at that time)

Reply:
I actually spent a good deal of time using nestjs decorators for Swagger. However, it's heavily coupled with the framework to the point I had a lot of dependencies. Also, putting them in devDependencies did not work at all even if I had all the necessary dependencies.
Also, a common package should only include functionality used by both app and API. I would argue against putting swagger schema since it is not being used in the FE.

Reply:
Check line 33. We can use a `ClassSerializer` to limit what can be returned. This can be perfected later.